devtools/server/actors/replay/connection.js

```diff
@@ -480,11 +480,19 @@ function getRecordingBrowser(key) {
   return key.ownerElement;
 }
 
+function initRecordingState(key, state) {
```

Comment:
I realize the `init` in this name of this is a bit confusing, maybe `updateRecordingState` would be better?

Reply:
maybe `makeRecordingStateObject` or something? `update` implies an action whereas this is just creating the object.

Reply:
The fact that it reads the existing value to create a new one is what seems worth capturing in the name to me. `makeRecordingStateObject` also glosses over that.

Reply:
ah, i see. Fair enough